src/nvim/screen.c

```diff
@@ -2750,7 +2759,9 @@ win_line (
           }
           p_extra = NULL;
           c_extra = ' ';
-          n_extra = get_breakindent_win(wp, ml_get_buf(wp->w_buffer, lnum, FALSE));
+          c_final = NUL;
+          n_extra = get_breakindent_win(wp,
+            ml_get_buf(wp->w_buffer, lnum, false));
```

Comment:
This line should be indented under 'w' in 'wp':
~~~
+          n_extra = get_breakindent_win(wp,
+                                                            ml_get_buf(wp->w_buffer, lnum, false));
~~~
but since that brings the issue of line length, perhaps indenting 2 spaces after `get_breakindent_win` might be better?
~~~
+          n_extra = get_breakindent_win(wp,
+                                                            ml_get_buf(wp->w_buffer, lnum, false));
~~~
Either way, no big deal.